docs/site/content/posts/08-build-deploy-apps-w-tce.md

```diff
@@ -0,0 +1,694 @@
+---
+title: "Build and Deploy an Application with VMware Tanzu Community Edition"
+slug: build-deploy-application-tanzu-community-edition
+date: 2022-04-28
+author: Cora Iberkleid
+image: /img/deploy-app-post-img10.png
+excerpt: "I recently presented a demo that shows how to use Tanzu Community Edition to automate many of the steps involved in cloud native services delivery. You can watch the demo here, or if reading works better than watching for you, then check out the transcript following!"
+tags: ['Cora Iberkleid']
+---
+I recently presented a demo that shows how to use VMware Tanzu Community Edition to automate many of the steps involved in cloud native services delivery.  You can watch the demo here, or if reading works better than watching for you, then check out the transcript following!
+
+<!-- https://gohugo.io/content-management/shortcodes/#youtube -->
+{{< youtube id="KdR_QGXdCzw" title="Build and Deploy an Application with VMware Tanzu Community Edition" >}}
+
+## Introduction
+
+In this demo, we’re going to use Tanzu Community Edition to build and deploy an application. We are using Tanzu Community Edition in a few different ways:
+
+1. To provision a Kubernetes cluster
+2. To enhance that cluster with additional tooling for building, publishing, and storing container images, and for managing applications at runtime
+3. To actually run our application
+
+Let’s get started.
+
+## Cluster provisioning
+
+You can see I actually have several clusters here:
+
+```shell
+$ kubectx
+kind-tce-demo
+tce-mgmt-cluster-admin@tce-mgmt-cluster
+tce-workload-cluster-1-admin@tce-workload-cluster-1
+```
+
+All of these were provisioned with Tanzu Community Edition. This first one (kind-tce-demo) is an example of an unmanaged cluster. This particular one happens to be running on Docker, locally on my machine, similar to how you might run kind or minikube.
+
+The next one is a management cluster (tce-mgmt-cluster), which is useful for use cases where you need to provision many clusters. I’ve used this management cluster, in fact, to provision the third “workload” cluster that you see here (tce-workload-cluster-1). These happen to be running on AWS.
+
+You can see by the color of the output that my context is pointing to the workload cluster. This is the one I will be using for this demo.
```

Comment:
Hmm, probably should have a screenshot if the color matters, since this code block isn't going to have any special color formatting.